### PR TITLE
Flag and suggest alternatives for common mispellings

### PR DESCRIPTION
**Pull request by aduth**
## 🛠 Summary of changes

Updates spell checker to consider common misspellings of words, including a suggested alternative.

**Draft:**

- [ ] Fix existing issues
- [ ] Any other to flag?

## 📜 Testing Plan

Verify build passes, and that revised content changes look sensible.

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -57,6 +57,14 @@
     "FICAM",
     "FPKI"
   ],
+  "flagWords": [
+    "Github->GitHub",
+    "Gitlab->GitLab",
+    "pki->PKI",
+    "Idp->IdP",
+    "idp->IdP",
+    "IDP->IdP"
```
Comment (mitchellhenke) [inline cspell.json:66]:
```suggestion
    "IDP->IdP",
    "Login->Login.gov"
```

**Review by mitchellhenke — APPROVED** [review]

File: cspell.json (hunk 1 above)
Comment (zachmargolis) [reply cspell.json:66]: Will `"login.gov->Login.gov"` flag URLs?

Comment (zachmargolis) [inline cspell.json:66]: what about making all-caps "IDP" the preferred spelling?

Comment (mitchellhenke) [reply cspell.json:66]: Good question, it looks like punctuation is ignored, so it can't detect `login.gov` anyway

Comment (mitchellhenke) [reply cspell.json:66]: https://cspell.org/docs/how-it-works/#things-to-note

Comment (aduth) [reply cspell.json:66]: Yeah, unfortunately "Login" catches correctly-spelled "Login.gov" as well because of the dot.

Comment (zachmargolis) [reply cspell.json:66]: dang! That is a bummer, but I guess it makes sense to match the industry

**Pull request merged**